flake.nix

```diff
@@ -16,5 +18,7 @@
         makeTest = import (pkgs.path + "/nixos/tests/make-test-python.nix");
         eval-config = import (pkgs.path + "/nixos/lib/eval-config.nix");
       };
-  };
+  }
+  // lib # shorthand
+  ;
```

Comment:
I think this is a bit to much. but we probably also want the stuff from ./default.nix in here?

Reply:
Yeah, I agree. After `module` is merged, the panorama looks a bit different.

I'll close and eventually re-open, in the mesure ot makes sense.